openwisp_users/apps.py

```diff
@@ -130,6 +137,22 @@ def connect_receivers(self):
             dispatch_uid='make_first_org_user_org_owner',
         )
 
+    @classmethod
+    def handle_organization_update(cls , instance , sender,**kwargs)  :
+        Organization = load_model("openwisp_users" , "Organization")
+        OrganizationUsers = load_model("openwisp_users" ,"OrganizationUser")
+        try:
+            old_obj = Organization.objects.get(pk=instance.pk)
+        except ObjectDoesNotExist:
+            logger.warning(f"Organization with pk {instance.pk} does not exist.")
+            return
+        except Exception as e:
+            logger.exception(f"An error occurred while fetching the organization: {e}")
+            return
+        if getattr(old_obj , "is_active") != getattr(instance , "is_active") :
+            for user in OrganizationUsers.objects.filter(organization = instance):
```

Comment:
We must not load the entire set in memory as it can crash big installations, to do that we use the `iterator()` method of the Django ORM, eg:
```suggestion
            for user in OrganizationUsers.objects.filter(organization = instance).iterator():
```